### DISCUSSION TITLE
https://www.xiaogenban1993.com/blog/24.06/enjoy_%E7%AC%94%E8%AE%B0%E7%BD%91%E7%AB%99%E6%8D%A2%E6%96%B0%E7%9A%AE%E8%82%A4%E5%95%A6

### DISCUSSION THREAD

**Discussion by giscus** [sunwu51/notebook#50] [General]
# https://www.xiaogenban1993.com/blog/24.06/enjoy_%E7%AC%94%E8%AE%B0%E7%BD%91%E7%AB%99%E6%8D%A2%E6%96%B0%E7%9A%AE%E8%82%A4%E5%95%A6

Generated by FreeCoder

https://www.xiaogenban1993.com/blog/24.06/enjoy_%E7%AC%94%E8%AE%B0%E7%BD%91%E7%AB%99%E6%8D%A2%E6%96%B0%E7%9A%AE%E8%82%A4%E5%95%A6

**Comment by sunwu51** [comment]
这是使用`giscus`平台，集成的`github discussion`来实现的评论区，完全托管不需要自己部署任何资源。

**Reply by sunwu51** [reply]
代码样式，因为giscus是iframe，所以无法使用博客系统中自定义的代码风格。
```js
console.log(123)
```
